server/index.js

```diff
@@ -1,9 +1,16 @@
+
+require('dotenv').load();
 const express = require('express');
+const bodyParser = require('body-parser');
+const router = require('./routes');
 
-const PORT = process.env.PORT || 4000;
+const { PORT = 4000 } = process.env;
 
 const app = express();
 
+app.use(bodyParser.urlencoded({ extended: false }));
```

Comment:
Also include bodyParser.json() middleware so it can accept JSON data and is not limited to browsers.

Reply:
Okay, good idea.